corehq/apps/export/migrations/0011_defaultexportsettings_usecouchfiletypes.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.16 on 2021-01-12 17:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('export', '0010_defaultexportsettings'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='defaultexportsettings',
+            name='cases_filetype',
+            field=models.CharField(choices=[('csv', 'CSV (zip file)'),
+                                            ('xlsx', 'Excel 2007+'),
+                                            ('xls', 'Excel (older versions)')],
+                                   default='xlsx', max_length=25),
+        ),
+        migrations.AlterField(
+            model_name='defaultexportsettings',
+            name='forms_filetype',
+            field=models.CharField(choices=[('csv', 'CSV (zip file)'),
```

Comment:
Has any data been written using the previous `CHOICES`? If so then it won't know to change "CSV" to "csv", "ECEL_2007_PLUS" to "xlsx", etc., right?

Reply:
Oops, sorry I missed the sentence in your description where you said production has 0 entries. Ignore!